xml/System.Net/NetworkCredential.xml

```diff
@@ -79,6 +79,9 @@
 ## Remarks
  The <xref:System.Net.NetworkCredential> class is a base class that supplies credentials in password-based authentication schemes such as basic, digest, NTLM, and Kerberos. Classes that implement the <xref:System.Net.ICredentials> interface, such as the <xref:System.Net.CredentialCache> class, return <xref:System.Net.NetworkCredential> objects.
 
+> [!NOTE]
+> On Linux platforms, use of NTLM authentication may require installing the NTLM plugin for the GSSAPI layer used by the implemention. On Ubuntu distributions, the package is called `gss-ntlmssp`.
+        
```

Comment:
Can we also add note about `System.Net.Security.UseManagedNtlm` as option starting in 8.0? And perhaps for completeness mention that on Windows with NTLM & Negotiate (Kerberos) always depend on OS GSSAPI. 

Any other recommendations @filipnavara ?

Reply:
Added